docs/source/en/using-diffusers/other-formats.md

```diff
@@ -144,6 +151,8 @@ pipeline = DiffusionPipeline.from_pretrained(
 ).to("cuda")
 pipeline.scheduler = UniPCMultistepScheduler.from_config(pipeline.scheduler.config)
 ```
+<!--Repository Not Found for url: https://huggingface.co/api/models/andite/anything-v4.0.
+Please make sure you specified the correct `repo_id` and `repo_type`.-->
```

Comment:
Looks like the model was removed. I think we can update the code snippet to use `runwayml/stable-diffusion-v1-5`, and then remove this comment. cc @sayakpaul do you know of any other cool model we can replace this with?

Reply:
@apolinario

Reply:
@standardAI, lets update this section with the below. You can probably keep most of the text, and just update the code snippets. The updated image URL is https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/blueprint-lora.png.

```py
from diffusers import StableDiffusionXLPipeline
import torch

pipeline = StableDiffusionXLPipeline.from_pretrained(
    "Lykon/dreamshaper-xl-1-0", torch_dtype=torch.float16, variant="fp16"
).to("cuda")

# uncomment to download the safetensor weights
!wget https://civitai.com/api/download/models/168776 -O blueprintify.safetensors

pipeline.load_lora_weights("/content/blueprintify.safetensors", weight_name="blueprintify.safetensors")
prompt = "bl3uprint, a highly detailed blueprint of the empire state building, explaining how to build all parts, many txt, blueprint grid backdrop"
negative_prompt = "lowres, cropped, worst quality, low quality, normal quality, artifacts, signature, watermark, username, blurry, more than one bridge, bad architecture"

images = pipeline(
    prompt=prompt,
    negative_prompt=negative_prompt,
    generator=torch.manual_seed(0),
).images[0]
images
```

Reply:
Done.